cockatrice/src/chatview.cpp

```diff
@@ -374,9 +417,27 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
                 while (fullMentionUpToSpaceOrEnd.size());
             }
         }
+        else if (index == highlightWordFirstIndex)
+        {
+            // You have received a valid mention of custom word!!
+            int firstSpace = (message.indexOf(" ") == -1 ? message.size() : message.indexOf(" "));
+            mentionFormat.setBackground(QBrush(getCustomMentionColor()));
```

Comment:
These are words you want to be notified of, at least how I feel. I don't see why a user can't type "Hey Zach" or "Hey Zelda" and I wouldn't be notified. I'm fine with this being highlighted the same color, but I'm open for debate.